fetch.bs

```diff
@@ -4315,12 +4346,68 @@ steps:
+   <a for="fetch params">controller</a>'s <a for="fetch controller">full timing info</a> to
+   <var>fetchParams</var>'s <a for="fetch params">timing info</a>.
+
+   <li>
+    <p>Set <var>fetchParams</var>'s <a for="fetch params">controller</a>'s
+    <a for="fetch controller">report timing steps</a> to the following steps given a
+    <a for=/>global object</a> <var>global</var>:
 
-   <li><p>If <var>fetchParams</var>'s <a for="fetch params">process response end-of-body</a> is not
-   null, then <a>queue a fetch task</a> to run <var>fetchParams</var>'s
-   <a for="fetch params">process response end-of-body</a> given <var>response</var> with
+    <ol>
+     <li><p>If <var>fetchParams</var>'s <a for="fetch params">request</a>'s <a for=request>URL</a>'s
+     <a for=url>scheme</a> is not an <a>HTTP(S) scheme</a>, then return.
+
+     <li><p>Set <var>timingInfo</var>'s <a for="fetch timing info">end time</a> to the
+     <a>relative high resolution time</a> given <var>unsafeEndTime</var> and
+     <var>global</var>.
+
+     <li><p>Let <var>cacheState</var> be <var>response</var>'s <a for=response>cache state</a>.
+
+     <li><p>Let <var>bodyInfo</var> be <var>response</var>'s <a for=response>body info</a>.
+
+     <li>
+      <p>If <var>response</var>'s <a for=response>timing allow passed flag</a> is not set,
+      then set <var>timingInfo</var> to a the result of <a>creating an opaque timing info</a> for
+      <var>timingInfo</var>, set <var>bodyInfo</var> to a new <a for=/>response body info</a>, and
+      set <var>cacheState</var> to the empty string.
+
+      <p class=note>This covers the case of <var>response</var> being a <a>network error</a>.
+
+     <li><p>If <var>fetchParams</var>'s <a for="fetch params">request</a>'s
+     <a for=request>initiator type</a> is not null, then
+     <a for=/>mark resource timing</a> given <var>timingInfo</var>, <var>request</var>'s
+     <a for=request>URL</a>, <a for="fetch params">request</a>'s
+     <a for=request>initiator type</a>, <var>global</var>, <var>cacheState</var>, and
+     <var>bodyInfo</var>.
+    </ol>
+
+   <li>
+    <p>Let <var>processResponseEndOfBodyTask</var> be the following steps:
+
+    <ol>
+     <li><p>Set <var>fetchParams</var>'s <a for="fetch params">request</a>'s
+     <a for=request>done flag</a>.
+
+     <li><p>If <var>fetchParams</var>'s <a for="fetch params">process response end-of-body</a> is
+     non-null, then run <var>fetchParams</var>'s
+     <a for="fetch params">process response end-of-body</a> given <var>response</var>.
+
+     <li><p>If <var>fetchParams</var>'s <a for="fetch params">request</a>'s
+     <a for="request">initiator type</a> is non-null and <var>fetchParams</var>'s
+     <a for="fetch params">request</a>'s <a for=request>client</a> is non-null, then run
+     <var>fetchParams</var>'s <a for="fetch params">controller</a>'s
+     <a for="fetch controller">report timing steps</a> given <var>fetchParams</var>'s
```

Comment:
We need to give only one argument ("global") to `report timing steps`, right?

Reply:
Right, removing extra arg, good catch